### PR TITLE
PW-5531 Support for googlepay as an additional variant for paywithgoogle

### PR DESCRIPTION
**Pull request by zaiddreakh**
## Summary

Support for googlepay as an additional variant for paywithgoogle
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

File: src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js [hunk 1]
```diff
@@ -49,6 +49,9 @@ if (
   store.checkoutConfiguration.paymentMethodsConfiguration.paywithgoogle.configuration[
```
Comment (wboereboom) [inline src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js:49]: what's the use of having 'merchantIdentifier' in a separate const called `id`?

File: src/cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.js [hunk 2]
```diff
@@ -102,7 +102,7 @@ function createPaymentRequest(args) {
       }
     }
     // Add empty browserInfo for GooglePay
-    if (paymentMethodType === 'paywithgoogle') {
+    if (paymentMethodType === 'paywithgoogle' || paymentMethodType === 'googlepay') {
       paymentRequest.browserInfo = {};
```
Comment (maassenbas) [inline src/cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.js:106]: Is the browserInfo not yet included in the `state.data`?

Comment (zaiddreakh) [reply src/cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.js:106]: you're right it was included in v4.2.0 https://github.com/Adyen/adyen-web/pull/847

File: src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js (hunk 1 above)
Comment (zaiddreakh) [reply src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js:49]: for live Google Pay payments the ID is required

**Review by wboereboom — APPROVED** [review]

**Review by maassenbas — APPROVED** [review]

**Pull request merged**